server/test/integration/com/thoughtworks/go/config/GoConfigMigrationIntegrationTest.java

```diff
@@ -1462,6 +1462,292 @@ public void shouldCreateProfilesFromMultipleAgentConfigsAcrossPipelines_asPartOf
         assertThat(profiles.get(1), is(expectedAWSProfile));
     }
 
+    @Test
+    public void shouldCreateAViewElementInEachTemplateIfPipelineGroupAdminsExist_asPartOfMigration89() throws NoSuchMethodException, IllegalAccessException, InvocationTargetException {
+        String configXml = "<cruise schemaVersion='88'>"
+                + " <server serverId='dev-id'>"
+                + "     <security>"
+                + "         <roles>"
+                + "             <role name='role1'>"
+                + "                 <users>"
+                + "                     <user>jyoti</user>"
+                + "                     <user>duck</user>"
+                + "                 </users>"
+                + "             </role>"
+                + "             <role name='role2'>"
+                + "                <users>"
+                + "                     <user>ketan</user>"
+                + "                     <user>mahesh</user>"
+                + "                </users>"
+                + "             </role>"
+                + "         </roles>"
+                + "     </security>"
+                + " </server>"
+                + " <pipelines group='first'>"
+                + "     <authorization>"
+                + "         <admins>"
+                + "             <user>user1</user>"
+                + "             <role>role1</role>"
+                + "         </admins>"
+                + "     </authorization>"
+                + " </pipelines>"
+                + " <pipelines group='second'>"
+                + "     <authorization>"
+                + "         <admins>"
+                + "             <user>user2</user>"
```

Comment:
Can you change/add one of these users, to have a repeated user between the two groups and similarly a repeated role between the two groups, so that you can show that the final list has no duplicates?

Edit: I see you have a different test for it later (which doesn't test unique roles, though).